test/unit/product.js

```diff
@@ -802,5 +802,31 @@ describe('Product class', () => {
         title: "sloth / small"
       }, 1111);
     });
+
+    it('empty cart and add variant to cart are called when destination is checkout', async () => {
+      product.config.product.buttonDestination = 'checkout';
+      product.config.cart.popup = true;
+      sinon.stub(product.cart, 'close');
```

Comment:
does this stub need to exist?

Reply:
I'm going to leave it in, but actually assert that it was called.

Reply:
I guess my only question is, is this testing behaviour, or testing a known implementation?